pihole

```diff
@@ -160,7 +161,7 @@ Time:
           local str="Disabling blocking for ${tt} seconds"
           echo -e "  ${INFO} ${str}..."
           local str="Blocking will be re-enabled in ${tt} seconds"
-          nohup bash -c "sleep ${tt}; pihole enable" </dev/null &>/dev/null &
+          nohup bash -c "sleep ${tt}; ${PI_HOLE_BIN_DIR}/pihole enable" </dev/null &>/dev/null &
```

Comment:
Similar to above but in a different context: Would this still work when there would be spaces in `PI_HOLE_BIN_DIR` ?

Reply:
What are the other options for solving this if it is an issue?

Reply:
```suggestion
          nohup bash -c "sleep ${tt}; \"${PI_HOLE_BIN_DIR}\"/pihole enable" </dev/null &>/dev/null &
```